### ISSUE TITLE
Offer to help

### ISSUE THREAD

**Issue by ahundiak** [thomasgenter/MVC-for-web#1]
Been following the discussion on sitepoint.  I think you have a good start and encourage you to continue.  Let me know if you get bogged down in some of the implementation details.  

For example, composer will generate an autoload file for you.  It's good to understand how to use spl_autoload directly but after that there is not much point in making a custom version.

Request/Response object implementations can be a time sink as you move past the basics.  If need be I can help you with integrating an off the shelf Request/Response component.

I think you might also find that introducing a dependency injection container will help streamline configuration.

Whatever you need.  Keep up the good work.

**Comment by thomasgenter** [comment]
Thanks a lot for your cheerful words!

As I mentioned in the discussion on sitepoint, I think I got the concept of mvc quite wrong. But after digging a bit deeper into things, I still have the impression, mvc is not actual the right pattern for php applications.

With that said, let me state this repo is at its current state rather some arranged notes in form of code, while experimenting on finding an alternative to mvc (like so many others and probably much experienced programmers did). It has yet to be arranged and completed to become a at least partly functional prototype to try out – though I tidied up things a little today.

Therefore, and because of a personal ambition to know and to understand each line of code in the prototype, there are currently no external components integrated. That may change later on.

If I still have your attention, let me explain my current approach in a shortened form.

* The „framework“ (/vendor/ted/ted/…) should actually only provide Request and Response, a Router and a DI-Container (Provider). The conceptual part can be found in /app
* I still use the terms model, view and controller, but I explicitly do not follow the mvc concept
* The rules of my (current) concept could be shortly described as follows:
  * Requests get routed to Controllers
  * Controllers mainly fill/complete the Response
  * Controllers sometimes manipulate Models
  * Controllers may use Views to build the Response
  * Views fetch data from Models rather then get fed by Controller
  * Templates/TemplateEngines DO NOT EVER fetch data or trigger controllers

Quite important (in my eyes) is the last bullet point, as this is associated to my initial question in the discussion on sitepoint. A View may interact with other components of the Application, a template must not. For example, a View always knows which Sub-views it contains, so it can create these and deliver them as variables to a potential template.

Long story short: I would highly appreciate your opinion on this idea. I’ll try to keep up with this concept and be glad to ask your help when completing this bare-bones project.

**Comment by ahundiak** [comment]
I have long since given up trying to shoehorn request/response applications into mvc.  So I think you are on the right track.
